readme.md

```diff
@@ -231,5 +270,19 @@ Convert the following ActiveRecord sequence to Mongoose:
 
 ```js
 // Your answer...
+var andy = Instructor.findOne({name: 'Andy'} function(err, person){
+  if (err) {
+    console.log('No person found by that name')
+  }else{
+    console.log('The name is ' + person.name)
+  }
+})
+
+andy.wishlistItems.create({description: 'Resin Laying Deer Figurine, Gold'}, function(err,data){
```

Comment:
so close, but remember that mongoose is async, which means that this line runs *before* the `findOne`. Here's a correct way:

```js
Instructor.findOne({name: "Andy"), function(err, result){
  result.wishlist_items.push({description: "Resin Laying Deer Figurine, Gold");
  result.save();
});
```